driver_author_analysis.py

```diff
@@ -6,31 +6,31 @@
 from lib.input.mbox.keyword_clustering import generate_kmeans_clustering
 from lib.input.mbox.keyword_digest import generate_keyword_digest
 from lib.analysis.author.community import vertex_clustering
+from driver_path import driver_path_class
```

Comment:
import mlcatconfig
config = mlcatconfigconfig.Config()
config.read('mlcat.cfg')

Reply:
The same comments are applicable to all the driver files.